src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml

```diff
@@ -195,5 +197,12 @@
             <argument type="service" id="security.token_storage" />
             <argument type="service" id="security.encoder_factory" />
         </service>
+
+        <!-- Cache Warmers -->
+        <service id="security.cache_warmer.expression_cache_warmer" class="Symfony\Bundle\SecurityBundle\CacheWarmer\ExpressionCacheWarmer">
```

Comment:
If ExpressionLanguage is not available, this cache warmer should not be registered, because it would still try to instantiate `security.expression_language`.
We already have code in the DI extension removing the relevant services. This one must be added in the list (and a simpler way for maintenance could be to move all services related to ExpressionLanguage to a separate XML file, which could be loaded conditionally).

Reply:
Its done now here as well: https://github.com/symfony/symfony/pull/26660/commits/adab5cec0ec07598c3f66ced07857226a5c58404

If the ExpressionLanguage is not available then it fails when using `allow_if`. So we can safely remove the definition if there are no expressions and it should be fine?